### PR TITLE
Remove unused dependencies

### PR DESCRIPTION
**Pull request by wardpenney**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,10 +33,6 @@
     "test": "echo \"No test specified\"",
     "contrib": "gulp"
   },
-  "dependencies": {
-    "bourbon": "^4.2",
-    "node-sass": "^3.4"
```
Comment (tysongach) [inline package.json]: Neat still needs a Sass compiler, yeah?

Comment (tysongach) [inline package.json]: For reference, this was added in https://github.com/thoughtbot/neat/pull/418

**Comment by wardpenney** [comment]
@tysongach I think `glup-sass` has `node-sass": "^3.5.0-beta.1"` as a dependency.

**Comment by tysongach** [comment]
It does, but we shouldn’t force Gulp as a dependency. Also `glup-sass` is only a _dev_ dependency, and not a runtime dependency.

**Comment by wardpenney** [comment]
Ahh, but isn't this only a dev tool? Does this have an effect on the end user who uses the neat package?

**Comment by tysongach** [comment]
@wardpenney No, because we publish to npm and this `package.json` serves both needs.

**Comment by tysongach** [comment]
It’s similar to how we declare Sass as a runtime dependency in the gemspec: https://github.com/thoughtbot/neat/blob/master/neat.gemspec#L26

**Comment by tysongach** [comment]
@wardpenney I just rebased this branch off `neat-2.0.0`. Want to wrap this up and get it merged in? I’m still curious about removing `node-sass`…Neat _needs_ a Sass compiler at some point.

**Comment by tysongach** [comment]
I just rebased this off `neat-2.0.0` and added back `node-sass` as a runtime dependency. Does this look good, @wardpenney & @whmii?

**Comment by tysongach** [comment]
@whmii Good to merge?

**Comment by whmii** [comment]
Yep. Lem rebase again and we'll get this in. cutting alpha soon

**Pull request merged**